distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java

```diff
@@ -223,6 +223,18 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
         return options(junitBundles(),
                 // HACK: incorrect version exported to override hamcrest-core from exam
                 // feature which causes a split package issue for rest-assured
+                wrappedBundle(mavenBundle("org.apache.httpcomponents",
+                        "com.springsource.org.apache.httpcomponents.httpclient")
```

Comment:
Thanks for the clarification.

On Mon, Oct 19, 2015 at 9:55 AM Vina Martin <notifications@github.com>
wrote:

> In
> distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java
> <https://github.com/codice/ddf/pull/267#discussion_r42396386>:
>
> > @@ -223,6 +223,18 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
> >          return options(junitBundles(),
> >                  // HACK: incorrect version exported to override hamcrest-core from exam
> >                  // feature which causes a split package issue for rest-assured
> > +                wrappedBundle(mavenBundle("org.apache.httpcomponents",
> > +                        "com.springsource.org.apache.httpcomponents.httpclient")
>
> Yes, the springsource versions have already been made into bundles.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codice/ddf/pull/267/files#r42396386>.
>